README.md

```diff
@@ -5,3 +5,17 @@ from stdin and writes it to stdout. Can also be run as a web service by passing
 the `-addr` option.
 
 ![PARTY](/out.gif)
+
+# Install
+
+```
+$ go get github.com/disintegration/imaging
```

Comment:
just running `go get` in the project root should be enough, you don't have to actually enumerate all the dependencies.